fixtures/room/create_room_fixtures.py

```diff
@@ -25,17 +31,42 @@
                 "roomType": "Meeting",
                 "capacity": 4,
                 "floorId": 4,
+                "roomTags": [
+                  {
+                    "name": "Block-A",
+                    "description": "Block"
+                  }
+                ],
                 "imageUrl": "https://www.officelovin.com/wp-content/uploads/2016/10/andela-office-main-1.jpg"  # noqa: E501
             }
         }
     }
 }
 
+non_existing_tag_mutation = '''
+    mutation {
+        createRoom(
+            name: "Mbarara", roomType: "Meeting", capacity: 4, floorId: 4, officeId: 1,,
```

Comment:
@bevkololi good work, though I'm wondering why you added an extra comma here and on line 6.

Reply:
Thanks @dnuwa, let me fix this.